src/main/kotlin/io/snyk/plugin/ui/UIUtils.kt

```diff
@@ -383,7 +383,7 @@ fun getJBCefBrowserIfSupported () : Pair<JBCefBrowser?, String> {
         return null to ""
     }
     val cefClient = JBCefApp.getInstance().createClient()
-    val jbCefBrowser = JBCefBrowserBuilder().setClient(cefClient).setEnableOpenDevToolsMenuItem(true).build()
+    val jbCefBrowser = JBCefBrowserBuilder().setClient(cefClient).setEnableOpenDevToolsMenuItem(false).build()
```

Comment:
This is useful for debugging but we don't want it by default to be enabled. Most of our development has been done anyway